src/raqm.c

```diff
@@ -1009,6 +1009,69 @@ raqm_get_glyphs (raqm_t *rq,
   return rq->glyphs;
 }
 
+/**
+ * raqm_get_par_resolved_direction:
+ * @rq: a #raqm_t.
+ *
+ * Gets the resolved direction of the paragraph;
+ *
+ * Return value:
+ * The #raqm_direction_t specifying the resolved direction of text,
+ * or #RAQM_DIRECTION_DEFAULT.
```

Comment:
It should be documented that this gives meaningful output only after `raqm_layout()` have been called.

Reply:
Done